### DISCUSSION TITLE
Doesn't support for custom resolutions

### DISCUSSION THREAD

**Discussion by BatuhanAksoyy** [ppy/osu#27628] [Ideas] [answered]
So today I noticed that a big input lag while I'm playing the game. My monitor was set to 75 hertz but game was using 60. The reason for that is I have created that resolution from NVIDIA Control Panel and it wasn't exists in the resolution list. Normal osu! supports custom resolutions but lazer doesn't support that so I have to play at 60hz which is really bad. Is there anything that you can do about it?

**Comment by peppy** [comment, marked as answer]
wait for sdl update at some point in the future. if you serach, this has been reported countless times.
